CA_DataUploaderLib/IOconf/IOconfMap.cs

```diff
@@ -119,5 +119,7 @@ public BoardSettings BoardSettings
         };
 
         public override string ToString() => $"{BoxName} - {USBPort ?? SerialNumber}";
+
+        protected override void ValidateName(string name) { } // no validation
```

Comment:
do we have some other code in IOconfMap validating the box name format? the state field can be used in plugins, so it would be good if this also is forced to the valid range

Reply:
you're right. This needs validation. The tricky bit is that serial/port is initially set as the name (which is likely not a valid name), and then overwritten with the real name...

Reply:
done